test-framework/common/src/main/java/io/quarkus/test/common/PathTestHelper.java

```diff
@@ -21,13 +21,28 @@
 import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
 
+/**
+ * Maps between builder test and application class directories.
+ */
 public final class PathTestHelper {
-
-    private static final String TEST_CLASSES_FRAGMENT_MAVEN = File.separator + "test-classes";
-    private static final String CLASSES_FRAGMENT_MAVEN = File.separator + "classes";
-    private static final String TEST_CLASSES_FRAGMENT_GRADLE = "classes" + File.separator + "java" + File.separator + "test";
-    private static final String CLASSES_FRAGMENT_GRADLE = "classes" + File.separator + "java" + File.separator + "main";
+    private static final Map<String, String> TEST2MAIN_DIR_FRAGMENTS = new HashMap<>();
+    static {
+        // eclipse
+        TEST2MAIN_DIR_FRAGMENTS.put( //eclipse
```

Comment:
Do we need `bin` for Eclipse? AFAIK, it's following the Maven/Gradle conventions when importing a Maven/Gradle project?

Reply:
Hi, sorry for the delay in answering - life and work interfered

In my experience, it always uses bin/main, but I use a custom Gradle plugin at work (and run eclipse task manually at home).

So I just tried importing the quarkus-quickstarts/getting-started project with Eclipse's builtin Gradle support. And it also uses bin/main as output.

Makes sense, or Eclipse and Gradle would step on each other's toes.

I will adjust for the two other comments and resubmit.

Cheers!